backend/tests/integration_tests/payloads/payloads.json

```diff
@@ -92,12 +92,26 @@
             "message": "Configuration created successfully."
         }
     },
+    "test_get_config": {
+        "method": "GET",
+        "url": "/api/v1/config_definition/test_config/config/test",
+        "expected_status": 200
+    },
```

Comment:
Yes, we can definitely have response checks along with statuses, but the unit tests take care of the return checks/ errors for all functionalities. I understand having robust integration tests can help with documentation and proofreading the API, but it should be least of our priorities now since we're trying to land a MVP.